site/committers.html

```diff
@@ -338,6 +338,10 @@ <h2>Current Committers</h2>
       <td>Holden Karau</td>
       <td>Apple</td>
     </tr>
+    <tr>
+      <td>Attila Zsolt Piros</td>
+      <td>Cloudera</td>
+    </tr>
```

Comment:
Don't forget to update htmls too man :-)

Reply:
I am testing my check action ;)